range_queries/sparse_table.cpp

```diff
@@ -13,14 +13,15 @@
 */
 
 #include <vector>
+#include <cassert>
 #include <iostream>
 #include <algorithm>
 
 /**
  * @namespace range_queries
```

Comment:
conflicting namespace name id
The documentation here defines a `namespace range_queries` while the namespace in the code on line 24 is `SparseTable`. 

All the code in this folder(`range_queries`) is encapsulated under `namespaces range_queries`. To organize multiple functions in a file, you can always create a sub-namespace as such...
```cpp
/**
  * @namespace range_queries
  * @brief Range Queries algorithms
  */
namespace range_queries {
/**
  * @namespace SparseTable
  * @brief Functions for sparse table implementation
  */
namespace SparseTable{

    // add your structs, classes, functions, etc here 

} // namespace SparseTable
} // namespace range_queries
```

Reply:
I have updated namespace with `range_queries`. And this is the same across all the code like before. Is there anything I should address?